MM-30279 Added view and clear cache

rachelbritton opened the pull request:
Added topic for cache management and updated service management accordingly for 11.2.

rachelbritton committed:
MM-30279 Added view and clear cache

Added topic for cache management and updated service management accordingly

rbjarna commented on guides/admin-manual/admin_cache_management.rst:
```diff
@@ -0,0 +1,65 @@
+   .. image:: ../../images/_admin-cache-management.png
+      :width: 80%
+
+The entries in the DNS server's cache will be displayed in the modal.
```
Just a thought. I think that we have more often used the naming "dialog" instead of modal when talking about these smaller screens. Could you check?

rachelbritton replied:
Good catch! Thank you!

rbjarna replied:
Nice 👍

rachelbritton committed:
MM-30279. Updated terminology and fixed image path

Updated terminology to align with the rest of the documentation and fixed an error in the image path.
